packages/sysdig/data_stream/event/fields/fields.yml

```
@@ -0,0 +1,415 @@
+                      type: keyword
+                    - name: name
+                      type: keyword
+                    - name: privileged
+                      type: boolean
+                - name: ct
+                  type: group
+                  fields:
+                    - name: id
+                      type: keyword
+                    - name: name
+                      type: keyword
+                    - name: region
+                      type: keyword
+                    - name: request.functionname
+                      type: keyword
+                    - name: request.host
+                      type: keyword
+                    - name: shortsrc
+                      type: keyword
+                    - name: srcdomain
+                      type: keyword
+                    - name: srcip
+                      type: ip
+                    - name: user.accountid
+                      type: keyword
+                    - name: user.arn
+                      type: keyword
+                    - name: user.identitytype
+                      type: keyword
+                    - name: user.principalid
+                      type: keyword
+                    - name: user.value
+                      type: keyword
+                    - name: useragent
+                      type: keyword
+                - name: evt
+                  type: group
+                  fields:
+                    - name: arg.request
+                      type: keyword
+                    - name: dir
+                      type: keyword
+                    - name: res
+                      type: keyword
+                    - name: type
+                      type: keyword
+                - name: fd
+                  type: group
+                  fields:
+                    - name: directory
+                      type: keyword
+                    - name: name
+                      type: keyword
+                    - name: sip
+                      type: ip
+                    - name: sport
+                      type: long
+                    - name: type
+                      type: keyword
+                - name: group
+                  type: group
+                  fields:
+                    - name: gid
+                      type: keyword
+                    - name: name
+                      type: keyword
+                - name: proc
+                  type: group
+                  fields:
+                    - name: args
+                      type: keyword
+                    - name: cmdline
+                      type: keyword
+                    - name: cwd
+                      type: keyword
+                    - name: exepath
+                      type: keyword
+                    - name: hash.sha256
+                      type: keyword
+                    - name: name
+                      type: keyword
+                    - name: pcmdline
+                      type: keyword
```

```suggestion
                      type: match_only_text
```
or match the type used by [`process.command_line`](https://www.elastic.co/docs/reference/ecs/ecs-process#field-process-command-line) which is a multi-fields including this.

---

I'm not entirely sure why `process.name` is a multi-fields like that since it's (almost) always going to be short token, but given that it is, it is probably worth matching that and using a multi-fields for the fields here that semantically correspond to ECS fields that have that.